trace.go

```diff
@@ -137,6 +142,16 @@ func (e *traceExporter) flush() {
 // order to not lose any tracing data. Only call Stop once per exporter. Repeated calls
 // will cause panic.
 func (e *traceExporter) stop() {
+loop:
+	// drain the input channel to catch anything the loop might not
+	for {
+		select {
+		case span := <-e.in:
+			e.receiveSpan(span)
```

Comment:
Awesome, thanks.

Reply:
Thank you too for being the catalyst in making me think deeper! 🙇